docs/source/conf.py

```diff
@@ -29,7 +29,7 @@
 
 # If your documentation needs a minimal Sphinx version, state it here.
 #
-# needs_sphinx = '1.0'
+needs_sphinx = '2.4.4'
```

Comment:
this  is probably not needed as we already have the `requirements.txt` file. We should try to keep track on the minimal version in one single place